packages/app/src/cli/services/dev.ts

```diff
@@ -84,7 +85,7 @@ async function dev(options: DevOptions) {
   const tunnelPort = await getAvailableTCPPort()
 
   let tunnelClient: TunnelClient | undefined
-  if (!options.tunnelUrl) {
+  if (!options.tunnelUrl && !options.noTunnel) {
```

Comment:
Is this just a general fix that made its way into here?

Reply:
@amcaplan , yes, I could create a different PR for this little fix. When you run the `dev` command using the `--not-tunnel` flag, a tunnel is created anyway